htdocs/mrp/mo_list.php

```diff
@@ -244,7 +244,7 @@
 
 foreach ($search as $key => $val) {
 	if (array_key_exists($key, $object->fields)) {
-		if ($key == 'status' && $search[$key] == -1) {
+		if ($key == 'status' && ($search[$key] == -1) || ($search[$key] == "")) {
```

Comment:
The condition is mixing && and || in same level. I think you must add ( ) around the 2 test on $search

Reply:
u right. My feeling for php must grow....